### PR TITLE
Refactor "fix" / Improvments

### PR DESCRIPTION
**Pull request by MatiasPastori**
More refactor

NVG preset changes, NVG automatic switch in or out vehicle.

Added ShaderComponents to the presets, (now being interpreted by VEM too)

Clean up patchAssets

UI changes, animation, sounds, etc.

### CODE REVIEW

File: ext/Client/Presets/Vanilla/MP_001/NVG.lua [hunk 1]
```diff
@@ -90,7 +90,7 @@ return [[
     "OutdoorLight":{
         "Enable":"true",
         "SunRotationX":"0.0",
-        "SunRotationY":"180.0",
+        "SunRotationY": "180.0",
```
Comment (IllustrisJack) [inline ext/Client/Presets/Vanilla/MP_001/NVG.lua]: unnecessary?

File: ext/Client/Presets/Vanilla/MP_001/Night.lua [hunk 2]
```diff
@@ -77,7 +77,7 @@ return [[
 	"OutdoorLight":{
 		"Enable":"true",
 		"SunRotationX":"0.0",
-		"SunRotationY":"180.0",
+		"SunRotationY": "180.0",
```
Comment (IllustrisJack) [inline ext/Client/Presets/Vanilla/MP_001/Night.lua]: same here?

File: ext/Client/Presets/Vanilla/MP_003/NVG.lua [hunk 3]
```diff
@@ -90,7 +90,7 @@ return [[
     "OutdoorLight":{
         "Enable":"true",
         "SunRotationX":"0.0",
-        "SunRotationY":"180.0",
+        "SunRotationY": "180.0",
```
Comment (IllustrisJack) [inline ext/Client/Presets/Vanilla/MP_003/NVG.lua]: same here

File: ext/Client/Presets/Vanilla/MP_003/Night.lua [hunk 4]
```diff
@@ -77,7 +77,7 @@ return [[
 	"OutdoorLight":{
 		"Enable":"true",
 		"SunRotationX":"0.0",
-		"SunRotationY":"180.0",
+		"SunRotationY": "180.0",
```
Comment (IllustrisJack) [inline ext/Client/Presets/Vanilla/MP_003/Night.lua]: same here

File: ext/Client/Presets/Vanilla/MP_007/NVG.lua [hunk 5]
```diff
@@ -90,7 +90,7 @@ return [[
     "OutdoorLight":{
         "Enable":"true",
         "SunRotationX":"0.0",
-        "SunRotationY":"180.0",
+        "SunRotationY": "180.0",
```
Comment (IllustrisJack) [inline ext/Client/Presets/Vanilla/MP_007/NVG.lua]: same here

File: ext/Client/Presets/Vanilla/MP_007/Night.lua [hunk 6]
```diff
@@ -77,7 +77,7 @@ return [[
     "OutdoorLight":{
        	"Enable":"true",
        	"SunRotationX":"0.0",
-		"SunRotationY":"180.0",
+		"SunRotationY": "180.0",
```
Comment (IllustrisJack) [inline ext/Client/Presets/Vanilla/MP_007/Night.lua]: what happened to all of those? :D

File: ext/Client/Presets/Vanilla/MP_011/Night.lua [hunk 7]
```diff
@@ -77,7 +77,7 @@ return [[
     "OutdoorLight":{
        "Enable":"true",
        "SunRotationX":"0.0",
-       "SunRotationY":"180.0",
+       "SunRotationY": "180.0",
```
Comment (IllustrisJack) [inline ext/Client/Presets/Vanilla/MP_011/Night.lua]: -

File: ext/Client/Presets/Vanilla/MP_012/NVG.lua [hunk 8]
```diff
@@ -90,7 +90,7 @@ return [[
     "OutdoorLight":{
         "Enable":"true",
         "SunRotationX":"0.0",
-        "SunRotationY":"180.0",
+        "SunRotationY": "180.0",
```
Comment (IllustrisJack) [inline ext/Client/Presets/Vanilla/MP_012/NVG.lua]: -

File: ext/Client/Presets/Vanilla/MP_012/Night.lua [hunk 9]
```diff
@@ -77,7 +77,7 @@ return [[
    "OutdoorLight":{
       "Enable":"true",
       "SunRotationX":"0.0",
-      "SunRotationY":"180.0",
+      "SunRotationY": "180.0",
```
Comment (IllustrisJack) [inline ext/Client/Presets/Vanilla/MP_012/Night.lua]: -

File: ext/Client/Presets/Vanilla/MP_013/NVG.lua [hunk 10]
```diff
@@ -90,7 +90,7 @@ return [[
     "OutdoorLight":{
         "Enable":"true",
         "SunRotationX":"0.0",
-        "SunRotationY":"180.0",
+        "SunRotationY": "180.0",
```
Comment (IllustrisJack) [inline ext/Client/Presets/Vanilla/MP_013/NVG.lua]: -

File: ext/Client/Presets/Vanilla/MP_017/NVG.lua [hunk 11]
```diff
@@ -90,7 +90,7 @@ return [[
     "OutdoorLight":{
         "Enable":"true",
         "SunRotationX":"0.0",
-        "SunRotationY":"180.0",
+        "SunRotationY": "180.0",
```
Comment (IllustrisJack) [inline ext/Client/Presets/Vanilla/MP_017/NVG.lua]: -

File: ext/Client/Presets/Vanilla/MP_017/Night.lua [hunk 12]
```diff
@@ -77,7 +77,7 @@ return [[
     "OutdoorLight":{
         "Enable":"true",
         "SunRotationX":"0.0",
-        "SunRotationY":"180.0",
+        "SunRotationY": "180.0",
```
Comment (IllustrisJack) [inline ext/Client/Presets/Vanilla/MP_017/Night.lua]: -

File: ext/Client/Presets/Vanilla/MP_018/NVG.lua [hunk 13]
```diff
@@ -90,7 +90,7 @@ return [[
     "OutdoorLight":{
         "Enable":"true",
         "SunRotationX":"0.0",
-        "SunRotationY":"180.0",
+        "SunRotationY": "180.0",
```
Comment (IllustrisJack) [inline ext/Client/Presets/Vanilla/MP_018/NVG.lua]: -

File: ext/Client/Presets/Vanilla/MP_018/Night.lua [hunk 14]
```diff
@@ -77,7 +77,7 @@ return [[
     "OutdoorLight":{
         "Enable":"true",
         "SunRotationX":"0.0",
-        "SunRotationY":"180.0",
+        "SunRotationY": "180.0",
```
Comment (IllustrisJack) [inline ext/Client/Presets/Vanilla/MP_018/Night.lua]: -

File: ext/Client/Presets/Vanilla/MP_Subway/NVG.lua [hunk 15]
```diff
@@ -90,7 +90,7 @@ return [[
     "OutdoorLight":{
         "Enable":"true",
         "SunRotationX":"0.0",
-        "SunRotationY":"180.0",
+        "SunRotationY": "180.0",
```
Comment (IllustrisJack) [inline ext/Client/Presets/Vanilla/MP_Subway/NVG.lua]: -

File: ext/Client/Presets/Vanilla/MP_Subway/Night.lua [hunk 16]
```diff
@@ -77,7 +77,7 @@ return [[
     "OutdoorLight":{
         "Enable":"true",
         "SunRotationX":"0.0",
-        "SunRotationY":"180.0",
+        "SunRotationY": "180.0",
```
Comment (IllustrisJack) [inline ext/Client/Presets/Vanilla/MP_Subway/Night.lua]: -

**Comment by MatiasPastori** [comment]
@IllustrisJack You were right, I messed up with that extra space ( just me editing multiple files at once)

File: ext/Client/Presets/Vanilla/MP_007/Night.lua (hunk 6 above)
Comment (MatiasPastori) [reply ext/Client/Presets/Vanilla/MP_007/Night.lua]: Already cleaned that up in the last commit

**Pull request merged**